### PR TITLE
Typo AKE -> AKS

### PR DESCRIPTION
**Pull request by EwertonJordao**

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -53,4 +53,4 @@ Get started with our [open source contributing guide](https://github.com/opencos
 
 We are actively looking for new contributors if you or your company are interested in getting involved – join us using the links below-
 
-Contact us via email (<support@kubecost.com>) or join us on [CNCF Slack](https://slack.cncf.io/) in the [#opencost](https://cloud-native.slack.com/archives/C03D56FPD4G) channel if you have questions!
\ No newline at end of file
+Contact us via email (<support@kubecost.com>) or join us on [CNCF Slack](https://slack.cncf.io/) in the [#opencost](https://cloud-native.slack.com/archives/C03D56FPD4G) channel if you have questions!
```
Comment (nikovacevic) [inline docs/FAQ.md:56]: What changed here?

**Review by nikovacevic — COMMENTED** [review]
AKS looks good. Just wondering what changed in the other line.

File: docs/FAQ.md (hunk 1 above)
Comment (EwertonJordao) [reply docs/FAQ.md:56]: I don't change this 👀

Comment (nikovacevic) [reply docs/FAQ.md:56]: Maybe a newline. No problem.

**Review by nikovacevic — APPROVED** [review]

**Pull request merged**